examples/dashboard.yml

```
@@ -0,0 +1,121 @@
+sources:
+  nyc_taxi:
+    description: Large version of nyc taxi dataset
+    driver: parquet
+    args:
+      urlpath: '{{ CATALOG_DIR }}/data/nyc_taxi_wide.parq'
+      columns: ['dropoff_x','dropoff_y','pickup_x','pickup_y',
+                'dropoff_hour','pickup_hour','passenger_count']
+    metadata:
+      fields:
+        dropoff_hour:
+          label: Drop-off Hour
+        pickup_hour:
+          label: Pick-up Hour
+        fare_amount:
+          label: Fare
+          unit: $
+        passenger_count:
+          label: Passenger Count
+        counts:
+          label: Ride counts
+      plot:
+        xlim: !!python/tuple [-8240227.037, -8231283.905]
+        ylim: !!python/tuple [4974203.152, 4979238.441]
+        kind: points
```

If you need to have access to other columns later, then we can include them in the original points plot

```suggestion
        kind: points
        hover_cols: ['dropoff_hour', 'pickup_hour', 'passenger_count']
```